src/PostgREST/ApiRequest.hs

```diff
@@ -208,13 +208,20 @@ userApiRequest schema rootSpec req reqBody
       "DELETE"  -> ActionDelete
       "OPTIONS" -> ActionInfo
       _         -> ActionInspect{isHead=False}
+  schema = case lookupHeader "Accept-Version" of
+             Nothing      -> case schemas of
+                               []                  -> ""
```

Comment:
An empty schema list shouldn't be possible. Maybe you could change the `configSchema` type to a [Data.List.NonEmpty](http://hackage.haskell.org/package/base-4.12.0.0/docs/Data-List-NonEmpty.html#t:NonEmpty).

Reply:
Yes the config parser makes sure this isn't possible. I wasn't aware of this type that's why I put the empty list case so the compiler would't complain. Thanks for introducing this, it is nice to be able to be this precise.